src/katello/deb_tracer.py

```diff
@@ -27,45 +27,42 @@ def use_flag():
 
 
 def needrestart():
-    process = subprocess.Popen(["needrestart", "-b"], stdout=subprocess.PIPE)
-    output = process.communicate()
-    return output[0].decode().split("\n")
+    output = subprocess.check_output(["needrestart", "-b"])
+    return output.decode().split("\n")
```

Comment:
If you use `universal_newlines=True` in `check_output`, I think you don't need to decode it.